packages/react-native-reanimated/src/layoutReanimation/web/transition/EntryExit.web.ts

```diff
@@ -0,0 +1,52 @@
+'use strict';
+import type { AnimationData, TransitionData } from '../animationParser';
+import { AnimationsData } from '../config';
+
+export function EntryExitTransition(
+  name: string,
+  transitionData: TransitionData
+) {
+  const exitingAnimationData = structuredClone(
+    AnimationsData[transitionData.exiting]
+  );
+  const enteringAnimationData = structuredClone(
+    AnimationsData[transitionData.entering]
+  );
+
+  const additionalData = {
+    translateX: `${transitionData.translateX}px`,
+    translateY: `${transitionData.translateY}px`,
+    scale: `${transitionData.scaleX},${transitionData.scaleY}`,
+  };
+
+  const keyframeData: AnimationData = {
+    name,
+    style:
+      '0' in exitingAnimationData.style
+        ? {}
+        : { 0: { transform: [additionalData] } },
+    duration: 300,
+  };
+
+  for (const [timestamp, styles] of Object.entries(
+    exitingAnimationData.style
+  )) {
+    if (styles.transform !== undefined) {
+      styles.transform.unshift(additionalData);
+    } else {
+      styles.transform = [additionalData];
+    }
+
+    const newTimestamp = parseInt(timestamp) / 2;
+
+    keyframeData.style[`${newTimestamp === 50 ? 49 : newTimestamp}`] = styles;
```

Comment:
`Math.min()`

Reply:
Done in [d571951](https://github.com/software-mansion/react-native-reanimated/pull/6232/commits/d571951851d72110a4c51a7b3f44dfc393240fe0)